lighthouse-core/audits/dobetterweb/has-inspector-issues.js

```diff
@@ -0,0 +1,198 @@
+ */
+
+'use strict';
+
+/** @typedef {{url: string}} IssueSubItem */
+/** @typedef {{issueType: string|LH.IcuMessage, subItems: Array<IssueSubItem>}} IssueItem */
+
+const Audit = require('../audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on various types of issues with the page. This descriptive title is shown to users when no issues were logged into the Chrome DevTools Issues panel. */
+  title: 'No issues in the `Issues` panel in Chrome Devtools',
+  /** Title of a Lighthouse audit that provides detail on various types of issues with the page. This descriptive title is shown to users when issues are detected and logged into the Chrome DevTools Issues panel. */
+  failureTitle: 'Issues were logged in the `Issues` panel in Chrome Devtools',
+  /* eslint-disable max-len */
+  /** Description of a Lighthouse audit that tells the user why issues being logged to the Chrome DevTools Issues panel are a cause for concern and so should be fixed. This is displayed after a user expands the section to see more. No character length limits. */
+  description: 'Issues logged to the `Issues` panel in Chrome Devtools indicate unresolved problems. They can come from network request failures, insufficient security controls, and other browser concerns. Open up the Issues panel in Chrome DevTools for more details on each issue.',
+  /* eslint-enable max-len */
+  /** Table column header for the type of issue. */
+  columnIssueType: 'Issue Type',
+  /** The type of an Issue in Chrome DevTools when a resource is blocked due to receiving a rejection in the response to a request made on the page. */
+  issueTypeBlockedByResponse: 'Blocked By Response',
+  /** The type of an Issue in Chrome DevTools when a site has large ads that use up a lot of the browser's resources. */
+  issueTypeHeavyAds: 'Heavy Ads',
+};
+
+const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
+
+class IssuesPanelEntries extends Audit {
+  /**
+   * @return {LH.Audit.Meta}
+   */
+  static get meta() {
+    return {
+      id: 'has-inspector-issues',
+      title: str_(UIStrings.title),
+      failureTitle: str_(UIStrings.failureTitle),
+      description: str_(UIStrings.description),
+      requiredArtifacts: ['InspectorIssues'],
+    };
+  }
+
+  /**
+   * @param {Array<LH.Crdp.Audits.MixedContentIssueDetails>} mixedContentIssues
+   * @return {LH.Audit.Details.TableItem}
+   */
+  static getMixedContentRow(mixedContentIssues) {
+    const requestUrls = new Set();
+    for (const issue of mixedContentIssues) {
+      const requestUrl = (issue.request && issue.request.url) || issue.mainResourceURL;
+      if (requestUrl) {
+        requestUrls.add(requestUrl);
+      }
+    }
+    return {
+      issueType: 'Mixed Content',
+      subItems: {
+        type: 'subitems',
+        items: Array.from(requestUrls).map(url => {
+          return {
+            url,
+          };
+        }),
```

Comment:
optional nit: can also be
```suggestion
        items: Array.from(requestUrls).map(url => ({url})),
```
here and below, but up do you if you find that more or less readable :)